### PR TITLE
Making public map bigger for mobile devices on landscape orientation

### LINKED ISSUE #3460: Fix landscape mode in the public map page
Map should take more space.

img_5859

### PR DESCRIPTION
**Pull request by xavijam**
When device changes to landscape, it will scroll to the map content and will take up all the space up to "cut" the map title.

screenshot_2015-05-04-18-41-50_720
image2
image1

Fixes #3460
And fixes the "jump" problem with fixed header in private dashboard.

cc @saleiva 
REVIEWER: @javierarce

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3444/))

File: lib/assets/javascripts/cartodb/public_map/public_map_window.js [hunk 1]
```diff
@@ -60,16 +60,20 @@ module.exports = cdb.core.View.extend({
     var windowHeight = this.$el.height();
     var mainInfoHeight = this.$body.find('.PublicMap-title').height() + 90 /* meta + margins */;
     var headerHeight = this.$body.find('.Header').height();
+    var landscapeMode = this.el.matchMedia && this.el.matchMedia("(orientation: landscape)").matches;
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_map/public_map_window.js]: Pure gold: http://davidwalsh.name/orientation-change

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3445/))

File: lib/assets/javascripts/cartodb/public_map/public_map_window.js (hunk 1 above)
Comment (javierarce) [inline lib/assets/javascripts/cartodb/public_map/public_map_window.js]: Nice. So this.el is `window` here?

**Comment by javierarce** [comment]
:+1:

**Comment by xavijam** [comment]
@saleiva, what do you think?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3456/))

**Comment by saleiva** [comment]
+1

File: lib/assets/javascripts/cartodb/new_public_dashboard/fav_map_view.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ module.exports = cdb.core.View.extend({
       legends:           false,
       time_slider:       false,
       loader:            false,
+      fullscreen:        false,
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb/new_public_dashboard/fav_map_view.js]: No fullscreen button in fav map.

File: lib/assets/javascripts/cartodb/public_map/public_map_window.js (hunk 1 above)
Comment (xavijam) [inline lib/assets/javascripts/cartodb/public_map/public_map_window.js]: Yep! :dancer:

**Pull request merged**